source/Nevermore/TableAliasGenerator.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Linq;
+
+namespace Nevermore
+{
+    public interface ITableAliasGenerator
+    {
+        string GenerateTableAlias(string tableName = null);
+    }
+
+    public class TableAliasGenerator : ITableAliasGenerator
+    {
+        int tableCount = 0;
+
+        public string GenerateTableAlias(string tableName = null)
+        {
+            // Return a predictable alias so we don't blow out SQL's query-plan cache.
+            tableCount++;
+            return $"ALIAS_{AliasLabel(tableName)}_{tableCount}";
```

Comment:
I can't find any tests that show the SQL generated by this

Reply:
We are mocking out the `ITableAliasGenerator` in `QueryBuilderFixture`. I presume this was done before because it used to generate random strings in some cases and we needed it to be predictable for the tests.

I'll change this to use the concerete `TableAliasGenerator` so that it covers this logic as well.

Reply:
Done